README.md

```diff
@@ -1,16 +1,10 @@
-# CMSIS Toolbox Installer
+# CMSIS-Toolbox Installer
 
-This is the installer for the CMSIS-Toolbox.
+This is the installer for the CMSIS-Toolbox. It provides command line tools for processing software packs that are provided in Open-CMSIS-Pack format. It is currently under development but supports already a wide range of use cases (refer to [**implementation progress**](./docs/progress.md)).
```

Comment:
The tools do not just process software packs but it is also to process projects and configure build systems to create target binaries.

Reply:
Updated REDME.md with more details.